.circleci/config.yml

```diff
@@ -41,6 +34,9 @@ commands:
       command:
         type: string
         default: 'build'
+      toolchainVersion:
```

Comment:
I'm thinking if we should back-port these (`toolchainVersion`, executor/job naming, etc.) to earlier branches.

Reply:
Maybe. I'm not sure if the added complexity is useful there or would make forward merges better or we'd just be making more work. If we think it's worth it, let's open a separate issue/PR for it.